packages/fake-data/src/fakeUsers.ts

```diff
@@ -193,4 +193,101 @@ export const fakeApplicants = (numToGenerate = 20): GeneratedUser[] => {
   return defaultGenerator(numToGenerate);
 };
 
+export const fakeUser = (): User => {
+  const departments = fakeDepartments();
```

Comment:
`GeneratedUser` type was not playing nice so I added this in for when you just want `User`